pisi/fetcher.py

```diff
@@ -180,19 +182,13 @@ def fetch (self):
 
         return self.archive_file
 
-    def _get_http_headers(self):
+    def _get_headers(self):
         headers = []
-        if self.url.auth_info() and (self.url.scheme() == "http" or self.url.scheme() == "https"):
+        if self.url.auth_info():
             enc = base64.encodestring('%s:%s' % self.url.auth_info())
-            headers.append(('Authorization', 'Basic %s' % enc),)
-        return tuple(headers)
-
-    def _get_ftp_headers(self):
-        headers = []
-        if self.url.auth_info() and self.url.scheme() == "ftp":
-            enc = base64.encodestring('%s:%s' % self.url.auth_info())
-            headers.append(('Authorization', 'Basic %s' % enc),)
-        return tuple(headers)
+            headers.append(('Authorization', 'Basic %s' % enc))
+        headers.append(('User-Agent', 'PiSi Fetcher/' + pisi.__version__))
```

Comment:
Maybe PiSi Fetcher should be changed to eopkg Fetcher

Reply:
Yep, that's what is was originally, copy pasta.